syntax = "proto3";
package traces;
import "google/protobuf/struct.proto";

service DatalakeManagerService {
    rpc InsertData (InsertRequest) returns (InsertResponse);
  }
  
  message InsertTraceRequest {
    string path = 1;
    google.protobuf.Struct data = 2;
  }
  
  message InsertResponse {
    string status = 1;
  }
